editor/import/3d/resource_importer_scene.cpp

```diff
@@ -1995,6 +2000,7 @@ void ResourceImporterScene::get_internal_import_options(InternalImportCategory p
 			r_options->push_back(ImportOption(PropertyInfo(Variant::INT, "generate/navmesh", PROPERTY_HINT_ENUM, "Disabled,Mesh + NavMesh,NavMesh Only"), 0));
 			r_options->push_back(ImportOption(PropertyInfo(Variant::INT, "physics/body_type", PROPERTY_HINT_ENUM, "Static,Dynamic,Area"), 0));
 			r_options->push_back(ImportOption(PropertyInfo(Variant::INT, "physics/shape_type", PROPERTY_HINT_ENUM, "Decompose Convex,Simple Convex,Trimesh,Box,Sphere,Cylinder,Capsule,Automatic", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_UPDATE_ALL_IF_MODIFIED), 7));
+			r_options->push_back(ImportOption(PropertyInfo(Variant::BOOL, "physics/backface_collision", PROPERTY_HINT_NONE, "Backface Collision"), 0));
```

Comment:
Would it be possible to add some code in `get_internal_option_visibility` that only makes it only show the `physics/backface_collision` option if `Trimesh` is selected in the `physics/shape_type` option. (There should be other examples of code doing something similar that you can copy for this.)